applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml

```diff
@@ -11,7 +11,7 @@ prompt-proto-service:
     tag: latest
 
   instrument:
-    pipelines: (survey="SURVEY")=[${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml]
+    pipelines: (survey="SURVEY")=[${PROMPT_PROCESSING_DIR}/pipelines/LATISS/ApPipe.yaml]
```

Comment:
The `${RUBIN_INSTRUMENT}` --> `LATISS` change seems irrelevant here.  Though I don't mind the change and we probably want it anyway.

Reply:
Sorry, I can at least split it into its own commit.